format/spec.md

```diff
@@ -177,20 +177,24 @@ A **`map`** is a collection of key-value pairs with a key type and a value type.
 | **`decimal(P,S)`** | Fixed-point decimal; precision P, scale S                                | Scale is fixed [1], precision must be 38 or less |
 | **`date`**         | Calendar date without timezone or time                                   |                                                  |
 | **`time`**         | Time of day without date, timezone                                       | Microsecond precision [2]                        |
-| **`timestamp`**    | Timestamp without timezone                                               | Microsecond precision [2]                        |
-| **`timestamptz`**  | Timestamp with timezone                                                  | Stored as UTC [2]                                |
+| **`timestamp`**    | Timestamp, with microsecond precision, without timezone                  | [4], [6]                                         |
+| **`timestamptz`**  | Timestamp, with microsecond precision, with timezone                     | [4], [7]                                         |
+| **`timestamp_ns`** | Timestamp, with nanosecond precision, without timezone                   | [5], [6]                                         |
+| **`timestamptz_ns`** | Timestamp, with nanosecond precision, with timezone                    | [5], [7]                                         |
 | **`string`**       | Arbitrary-length character sequences                                     | Encoded with UTF-8 [3]                           |
 | **`uuid`**         | Universally unique identifiers                                           | Should use 16-byte fixed                         |
 | **`fixed(L)`**     | Fixed-length byte array of length L                                      |                                                  |
 | **`binary`**       | Arbitrary-length byte array                                              |                                                  |
 
 Notes:
 
-1. Decimal scale is fixed and cannot be changed by schema evolution. Precision can only be widened.
-2. All time and timestamp values are stored with microsecond precision.
-    - Timestamps _with time zone_ represent a point in time: values are stored as UTC and do not retain a source time zone (`2017-11-16 17:10:34 PST` is stored/retrieved as `2017-11-17 01:10:34 UTC` and these values are considered identical).
-    - Timestamps _without time zone_ represent a date and time of day regardless of zone: the time value is independent of zone adjustments (`2017-11-16 17:10:34` is always retrieved as `2017-11-16 17:10:34`). Timestamp values are stored as a long that encodes microseconds from the unix epoch.
-3. Character strings must be stored as UTF-8 encoded byte arrays.
+1. `decimal(P,S)` scale (`S`) is fixed, and cannot be changed by schema evolution. Precision (`P`) can only be widened by schema evolution.
+2. `string` values must be stored as UTF-8 encoded byte arrays.
```

Comment:
There are some discrepancies here. `[2]` is referenced in the time field, which doesn't seem applicable for the `time` type. Can you keep the changes to a minimum? 🙏

Reply:
Yikes, you're right.

I've adjusted so that only `[2]` changes.